packages/graphql/src/schema-model/generate-model.ts

```diff
@@ -135,15 +135,10 @@ function getCypherTarget(schema: Neo4jGraphQLSchemaModel, attributeType: Attribu
     if (attributeType instanceof ObjectType) {
         const foundConcreteEntity = schema.getConcreteEntity(attributeType.name);
         if (!foundConcreteEntity) {
-            throw new Neo4jGraphQLSchemaValidationError(
-                `@cypher field must target type annotated with the @node directive${attributeType.name}, `
-            );
+            return undefined;
         }
         return schema.getConcreteEntity(attributeType.name);
     }
-    if (attributeType instanceof InterfaceEntity || attributeType instanceof UnionEntity) {
-        throw new Error("@cypher field target cannot be an interface or an union");
-    }
```

Comment:
Maybe we should be keeping this given the skipped test?

Reply:
This will fail on interfaces targeting nodes (which works properly now), this check is not executed on non node interfaces, keep in mind that those don't even reach the model

Reply:
These error checks cover the cases that now work. The skipped test is not covered by these cases (non-node interfaces do not really reach the Neo4jGraphQL model generation) 

The skipped test throw an error that is not covered by any of these cases, I'll see if it is possible to validate that case here to throw a better error